scss/atoms/inputs/_inputs.scss

```diff
@@ -0,0 +1,37 @@
+/*
+* ==========================================================================
+* Inputs
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"mixins";
+
+.input {
+	@include input();
+}
+
+.input-label {
+	@include input-label();
+}
+
+/* Input Sizes */
```

Comment:
Do we want any other input sizes? we have a larger variety of buttons. I could see smaller inputs being needed. Also the small size seems to be closer to the 'medium' button size.